providers/oracle/pyproject.toml

```diff
@@ -64,7 +64,7 @@ dependencies = [
 # Any change in the dependencies is preserved when the file is regenerated
 [project.optional-dependencies]
 "numpy" = [
-    "numpy",
+    "numpy>=1.26.0",
```

Comment:
Again - with isolation, I found we miss lower-binding here.

Reply:
Ah, yeah also had this in example dags, Python 3.12 needs 1.28 at least. Older Python accepts also older stuff.